src/frontend/src/flows/manage.ts

```diff
@@ -45,16 +54,44 @@ const style = () => html`<style>
     width: auto;
     margin: 0;
     cursor: pointer;
-    color: #387ff7;
+  }
+
+  .labelActionText {
+    float: right;
```

Comment:
Just FYI: I actually never use `float: right`. No particular reason.

I would probably have use `display: flex` on the container and `justify-content: space-between`.

Reply:
Done! thanks